client/gutenboarding/index.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * External dependencies
+ */
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import { createSite, main, redirectIfNotEnabled } from './controller';
+import { makeLayout, render as clientRender } from 'controller';
+import { hideMasterbar as hideMasterbarAction } from 'state/ui/actions';
+
+function hideMasterbar( context: PageJS.Context, next ) {
+	context.store.dispatch( hideMasterbarAction() );
+	next();
+}
+
+export default function() {
+	page( '/gutenboarding', redirectIfNotEnabled, hideMasterbar, main, makeLayout, clientRender );
+	page(
+		'/gutenboarding/onboard',
```

Comment:
This route is currently giving me `We're sorry, but an unexpected error has occurred` :confused:

Reply:
I think that's coming from `import { Header } from '@wordpress/edit-post';`